packages/react-components/src/helpers/utilities.js

```diff
@@ -0,0 +1,48 @@
+export const uniqueId = (
+  counter =>
+  (str = '') =>
+    `${str}${++counter}`
+)(0);
```

Comment:
I'm a little wary about this, but if we're going to have it I think we should at least have unit tests for it.